subgraph/networks/bsc.subgraph.yaml

```diff
@@ -74,27 +74,47 @@ dataSources:
     name: MessengerRegistry
     network: bsc
     source:
-      address: '0x837Def7a9AA8FC9480d3f5bC1D61aF276eEEfE73'
+      address: "0x837Def7a9AA8FC9480d3f5bC1D61aF276eEEfE73"
       startBlock: 11398530
       abi: MessengerRegistry
     mapping:
       kind: ethereum/events
-      apiVersion: 0.0.4
+      apiVersion: 0.0.5
       language: wasm/assemblyscript
       entities:
         - MessengerRegistry
         - IMessenger
       abis:
         - name: MessengerRegistry
-          file: ../abis/@stacktical/dsla-contracts/contracts/MessengerRegistry.sol/MessengerRegistry.json
+          file: ../abis/MessengerRegistry.json
         - name: IMessenger
-          file: ../abis/@stacktical/dsla-contracts/contracts/interfaces/IMessenger.sol/IMessenger.json
+          file: ../abis/IMessenger.json
       eventHandlers:
         - event: MessengerRegistered(indexed address,indexed address,string,uint256,uint256)
           handler: handleMessengerRegistered
         - event: MessengerModified(indexed address,indexed address,string,uint256,uint256)
           handler: handleMessengerModified
       file: ../mapping.ts
+  - kind: ethereum/contract
+    name: PeriodRegistry
+    network: bsc
+    source:
+      address: "0x9149aeB2b9a731eC0417Ef5587b5175De2E77566"
+      startBlock: 11398522
+      abi: PeriodRegistry
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.5
+      language: wasm/assemblyscript
+      entities:
+        - PeriodRegistry
+      abis:
+        - name: PeriodRegistry
+          file: ../abis/PeriodRegistry.json
+      eventHandlers:
+        - event: PeriodModified(uint8,uint256)
+          handler: handlePeriodModified
+      file: ../mapping.ts
```

Comment:
Why is this back?
Let's review all yaml to make sure it's not.

Reply:
removed